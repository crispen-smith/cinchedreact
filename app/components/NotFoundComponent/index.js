/**
 *
 * NotFoundComponent
 *
 */

import React from 'react';
// import PropTypes from 'prop-types';
import Not from '../Not';

/* eslint-disable react/prefer-stateless-function */
class NotFoundComponent extends React.PureComponent {
  render() {
    return (
      <Not>
        Looks like the page you were looking for celebrated a litle too hard and
        has disapeared.
      </Not>
    );
  }
}

NotFoundComponent.propTypes = {};

export default NotFoundComponent;
