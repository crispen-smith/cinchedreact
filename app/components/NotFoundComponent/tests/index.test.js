import React from 'react';
import renderer from 'react-test-renderer';

import NotFoundComponent from '../index';

describe('<NotFoundComponent />', () => {
  it('Renders a NotFoundComponent', () => {
    const tree = renderer.create(<NotFoundComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
