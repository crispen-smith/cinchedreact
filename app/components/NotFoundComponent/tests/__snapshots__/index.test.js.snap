// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundComponent /> Renders a NotFoundComponent 1`] = `
<div
  className="Not-bJJsFZ hKjuvT"
>
  Looks like the page you were looking for celebrated a litle too hard and has disapeared.
</div>
`;
